docs/cli.md

```diff
@@ -319,6 +319,16 @@ viam data database configure --org-id=abc --password=my_password123
 viam data database hostname --org-id=abc
 ```
 
+Viam currently only supports deleting approximately 500 files at a time.
+To delete more data iterate over the data with a shell script:
+
+```sh
+# deleting one hour of image data
+for i in {00..59}; do
+viam data delete --org_ids=<org-id> --data_type=binary --mime_types=image/jpeg,image/png --start=2022-12-13T06:00:00.000Z --end=2022-12-13T06:${i}:00.000Z
+done
```

Comment:
The latest CLI versions uses slightly different syntax:

```suggestion
for i in {00..59}; do
  viam data delete binary --org-ids=c17f4c9e-3026-492b-81fb-a0d60f17b54a --mime-types=image/jpeg,image/png --start=2024-05-13T11:00:00.000Z --end=2024-05-13T11:${i}:00.000Z;
done
```

But I'm still getting errors with parsing the variable in the end time:

```
Error: Could not parse end flag: parsing time "2024-05-13T11:0:00.000Z" as "2006-01-02T15:04:05Z07:00": cannot parse "0:00.000Z" as "04"
Error: Could not parse end flag: parsing time "2024-05-13T11:1:00.000Z" as "2006-01-02T15:04:05Z07:00": cannot parse "1:00.000Z" as "04"
Error: Could not parse end flag: parsing time "2024-05-13T11:2:00.000Z" as "2006-01-02T15:04:05Z07:00": cannot parse "2:00.000Z" as "04"
```

It's interpolating properly, but somehow several decades off? Looking into it.

Reply:
I'm just trying and it's working for me with what you've but in place

Reply:
Maybe it's a TZ setting for my shell. I switched to `ksh` (macOS default) and you're right, it works just fine.

LGTM!